addons/crm/wizard/crm_module_access_request.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models
+
+
+class CrmModuleAccessRequest(models.TransientModel):
+    _name = 'crm.module.access.request'
+    _inherit = 'base.module.install.request'
+    _description = 'Module Access Request'
+
+    def action_send_request(self, template_id='base_install_request.mail_template_base_install_request'):
+        super().action_send_request(template_id='crm.mail_template_crm_access_request')
```

Comment:
Should update the default parameter value instead, otherwise the parameter is useless

But also, looking at it, I don't really see how the templates are different. Do we really need two?